app/src/main/java/com/example/happybirthday/MainActivity.kt

```diff
@@ -37,25 +48,57 @@ class MainActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colorScheme.background
                 ) {
-                    Greeting("Android")
+                    BirthdayGreetingWithImage( stringResource(R.string.happy_birthday_text),
+                        stringResource(R.string.signature_text)
+                    )
                 }
             }
         }
     }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
-        modifier = modifier
-    )
+fun BirthdayGreetingWithText(message: String, from: String, modifier: Modifier = Modifier) {
+    // Create a column so that texts don't overlap
+    Column(
+        modifier = modifier.fillMaxSize(),
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Text(
+            text = message,
+            fontSize = 36.sp,
+            modifier = Modifier
```

Comment:
this actually fits on a single line

Reply:
We added new parameters 
`            lineHeight = 100.sp,
`